script/browserify.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
```

Comment:
I know about browserify and that's where the name came from. For some reason, the real browserify did not always work for me (lately) and since I am using heterarchy at work in the browser (where I could not get it to work with browserify) I wrote the little script.

I think the underscore lib writes the javascript in a way that browserification is not even necessary. Maybe that would be the best way. The question then would be how to define the namespaces.
I just recently created a repo called [JsUml](https://github.com/jneuendorf/JsUml) where it tried to do it like underscore (see https://github.com/jneuendorf/JsUml/blob/master/js_uml.coffee).